content/en/cluster.md

```diff
@@ -2,26 +2,22 @@
 title: Cluster
 status: Completed
 category: Concept
-tags: ["infrastructure", "fundamental", ""]
+tags: ["infrastructure", "fundamental", "architecture"]
 ---
 
-A cluster is a group of computers or applications that work together towards a common goal. 
-In the context of cloud native computing, the term is most often applied to [Kubernetes](/kubernetes/). 
-A Kubernetes cluster is a set of services (or workloads) that run in their own containers, usually on different machines. 
-The collection of all these [containerized](/containerization/) services, connected over a network, represent a cluster.
+A cluster is a group of computers or applications called [Nodes](/nodes/) that work together towards a common goal.
+In cloud native computing, the term is often applied when talking about a [Kubernetes](/kubernetes/) cluster.
+It can be seen as a specific kind of [distributed systems](/distributed-systems/) where the nodes are a bit more tightly coupled.
 
 ## Problem it addresses 
 
-Software that runs on a single computer presents a single point of failure 
-— if that computer crashes, or someone accidentally unplugs the power cable, 
-then some business-critical system may be taken offline. 
+Software that runs on a single computer presents a single point of failure — if that computer crashes, or someone accidentally unplugs the power cable, then some business-critical system may be taken offline. 
```

Comment:
For reviewers: no content update made here, just a semantic line break adjustment.

Reply:
I will have to be honest here and say that my understanding for semantic line break does match the old version but not the new version. What do you think @jihoon-seo @seokho-son @hlipsig?

As a reference: https://sembr.org saying a break should be included after commas.

Reply:
Oh, I may have a different understanding too. I was just going by what we have in [our style guide](https://glossary.cncf.io/style-guide/).

> "Please use semantic line breaks (one sentence per line) to ease review."

Happy to make updates if we want, but we should be clearer in our style guide if we do 🙂

Reply:
Maybe we open an issue to discuss -- in which case, no change would be needed here (ignore the suggestions). We can update later if we decide to make a change.

Reply:
Contrary to what our style guide states, "[semantic line breaks](https://sembr.org)" do not necessarily mean "one sentence per line". I'll open a PR to update that part of the style guide.